io.jor.nanosaur2.yml

```diff
@@ -0,0 +1,37 @@
+app-id: io.jor.nanosaur2
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: Nanosaur2
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: nanosaur2
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/jorio/Nanosaur2.git
+        #tag: v2.1.0 ### TODO: Uncomment this for next release - Can't use tag 'v2.1.0' because AppStream metadata files were added in a later commit
+        commit: 589931b7475d2efa8ee64febd874773f0003f236
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+      - type: file
+        path: nanosaur2_launcher.sh
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    no-make-install: true
+    build-commands:
```

Comment:
```suggestion
    post-install:
```

Reply:
Done in [684d9c0](https://github.com/flathub/flathub/pull/3885/commits/684d9c02a13e3a9cfaea9d9f8fce8096ba1a5190)